pkg/controller/importconfig/cluster_info.go

```diff
@@ -34,12 +34,15 @@ import (
 
 // getBootstrapSecret looks for the bootstrap secret from bootstrap sa
 func getBootstrapSecret(ctx context.Context, kubeClient kubernetes.Interface, managedCluster *clusterv1.ManagedCluster) (*corev1.Secret, error) {
+	secretNames := []string{getBootstrapSASecretName(managedCluster.Name)}
+
```

Comment:
this needs more doc, because it depends on a ocp specific function.

Reply:
add more doc, and this dose not depend on the ocp specific function, it only is related to kube version